src/screens/Sidebar/ReportBugs.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { View, WebView, ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+import { cannyBoardTokens } from 'kitsu/constants/app';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+class ReportBugs extends React.Component {
```

Comment:
Maybe we should abstract this into a general component so we can reuse it for each board? Also I think we're missing the database requests board.